bin/runner.js

```diff
@@ -200,6 +200,11 @@ var argv = yargs
           default: false,
           describe: Constants.cliMessages.RUN.SYNC_DESCRIPTION,
           type: "boolean"
+        },
+        'headed': {
+          default: true,
```

Comment:
The default should be set to false?

Reply:
When `headed`, headless should be false, i.e. `headed` will be true.